### DISCUSSION TITLE
VPN

### DISCUSSION THREAD

**Discussion by boggsdabeast** [Luctia/ezarr#33] [Q&A]
Safety first as protection is always good!

Just asking a dumb question on if docker gets around needing a VPN "requirement". Is it recommended to have either host machine and/or docker container (torrent) use VPN?

**Comment by Luctia** [comment]
If I'm not mistaken, it doesn't really matter. Having a VPN on the host machine will route all internet through a VPN, having a VPN container that the torrent container uses routes just the torrent traffic through the VPN.

**Comment by M-Davies** [comment]
https://github.com/qdm12/gluetun is pretty decent to connect containers to a VPN

**Reply by Luctia** [reply]
Yeah, I started working on this using gluetun a while ago, but I ran into some implementation issues since it supports such a large amount of providers. It's kinda hard to implement that into the CLI, but I would like to use gluetun.

**Reply by ngcheeyuan** [reply]
I was just thinking about this. Noticed that there weren't any VPN container. My current implementation is very simple and only runs torrent through gluetun. Looking forward to implementing the rest of the services.
